src/sections/changableBlock/ChangableBlock.jsx

```diff
@@ -0,0 +1,30 @@
+import './ChangableBlock.css'
+import { Portfolio } from '../portfolio'
+import { Info } from '../info'
+import { useState } from 'react'
+
+export const ChangableBlock = () => {
+    const [block, setBlock] = useState(<Info/>);
+    function handleCLick () {
+        block.type.name == 'Info' ? setBlock(<Portfolio/>):setBlock(<Info/>);
+        handleToggle();
+        handleText();
+    }
+
+    const [isActive, setActive] = useState("false");
```

Comment:
try to have state at top of the component
just `false`
looks like you don't need this state
you can detect active state from your prev state